### PR TITLE
Small fixes and enhancements

### PR DESCRIPTION
**Pull request by Bulwagga**
removed dependency on ovos username, fixed spellings in README.md, made edits to script user output to point to updated released documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 <strong>** DEV BRANCH **</strong>
 
-<strong>Images built from this branch are NOT guarenteed to always work.</strong>
+<strong>Images built from this branch are NOT guaranteed to always work.</strong>
 
-Run ovos on top of RaspberryPiOS
+Run OVOS on top of RaspberryPiOS
 
-Prebuilt images can be found on the [OpenVoiceOS Downloads Page](https://downloads.openvoiceos.com/images/raspbian/)
+Prebuilt images will be found on the [OpenVoiceOS Downloads Page](https://downloads.openvoiceos.com/images/raspbian/) once our image server is operational.
```
Comment (builderjer) [inline README.md]: I have a server setup for this now through the github actions.  The address will be 
https://ovosimages.ziggyai.online/raspbian

**Review by builderjer — CHANGES_REQUESTED** [review]
small change on image server

And if you can do the refactor to `README.md ` and `manual_user_install.sh`

**Comment by Bulwagga** [comment]
> small change on image server
> 
> And if you can do the refactor to `README.md ` and `manual_user_install.sh`

Happy to do it. Ziggi AI? :)

File: README.md (hunk 1 above)
Comment (Bulwagga) [reply README.md]: references to ziggy ai added to README.md, manual_user_install.sh, and install_rasbpian.md (in documentation)

**Comment by builderjer** [comment]
I am old.  I grew up on the TV show Quantum Leap.  The computer was named Ziggy.  Any ziggyai.online reference throughout any ovos code is my server.  And I am why the WW 'Hey Ziggy'.

**Review by builderjer — APPROVED** [review]

**Comment by builderjer** [comment]
@Bulwagga can you review [this file](https://github.com/OpenVoiceOS/raspbian-ovos/blob/rewrite/manual_user_install.sh) with the changes in the repo please.  I think I got everything updated

**Comment by Bulwagga** [comment]
Will do... It will take a day. Thanks

**Comment by Bulwagga** [comment]
The script looks reasonable.  I'd like to do a test build.  Which repository should I clone to test it?

**Comment by builderjer** [comment]
the `rewrite` branch of `raspbian-ovos`
https://github.com/OpenVoiceOS/raspbian-ovos/tree/rewrite

Also, I don't know if you are on matrix at all, but I would like to invite you to a couple of channels there.

https://matrix.to/#/#openvoiceos-development:matrix.org
https://matrix.to/#/#openvoiceos-general:matrix.org

There are several others, but this will get you started if you aren't already on there

**Comment by Bulwagga** [comment]
I need some help... What is the right command to clone the rewrite code? I thought the "Code/https" selector would somehow get me the right code, but it's not the right repository.  Thanks

**Comment by Bulwagga** [comment]
I'm using the .zip in the meantime. Pretty sure it's up-to-date.

**Comment by builderjer** [comment]
```
git clone https://github.com/OpenVoiceOS/raspbian-ovos

cd raspbian-ovos

git checkout rewrite
```

**Comment by builderjer** [comment]
and lets move this [here](https://github.com/OpenVoiceOS/raspbian-ovos/pull/61)

then I get notifications on it again

**Pull request merged**